beaglebone/servo/src/__main__.py

```diff
@@ -0,0 +1,79 @@
+import time
+import Adafruit_BBIO.PWM as PWM
+import lcm
+from rover_msgs import ServoCmd
+
+
+# Don't know if this is the same for the servo we're using
+SERVO_MAX_DC = 10.0
+SERVO_MIN_DC = 4.0

```

Comment:
Have these values been tested with the science box servos?

Reply:
this has not been tested...values from last year's code ISH has been looking for the servos